src/toil/job.py

```diff
@@ -1381,6 +1378,10 @@ class JobFunctionWrappingJob(FunctionWrappingJob):
     instance (see :func:`toil.job.Job.Run`), it is made a variable of the wrapping job \
     called fileStore.
     """
+    @property
+    def fileStore(self):
+        return self._fileStore
+
     def run(self, fileStore):
         userFunction = self._getUserFunction()
         self.fileStore = fileStore
```

Comment:
I had thought this was the agreed solution. Are you suggesting that I add the public fileStore property to the Job class?